lxd/storage/drivers/volume.go

```diff
@@ -285,6 +285,33 @@ func (v Volume) Snapshots(op *operations.Operation) ([]Volume, error) {
 	return snapVols, nil
 }
 
+// SnapshotsMatch checks that the snapshots according to the storage driver match those provided (although not
+// necessarily in the same order).
```

Comment:
This sounds a bit confusing :)

Reply:
The order bit?

Reply:
`checks that the snapshots according` doesn't sound like valid English to me ;)

Reply:
How about "checks that the snapshots list according"

Reply:
Or maybe its needs a couple of commas, e.g. "checks that the snapshots, according to the storage driver, match those provided"

E.g. https://dictionary.cambridge.org/grammar/british-grammar/according-to
"The government, according to a poll taken last month, may lose the next election"

Reply:
yep, the commas make my brain parse this properly now ;)

Reply:
Fixed